firmware/network/border_router/Kconfig

```diff
@@ -0,0 +1,18 @@
+menu "Border Router"
+    config ADDRESS_IPV6
+    string "Address IPV6"
+    default "2001:db8:2::1"
+
+    menu "INTERFACE"
+
+        choice INTERFACE_MODE
+            prompt "Select interface mode"
+            config WIRED_INTERFACE
+                bool "WIRED"
+
+            config WIRELESS_INTERFACE
+                bool "WIRELESS"
+        endchoice
```

Comment:
IMO the selection should not be a choice, e.g I want to set the settings to wired and Wireless interface in the first configuration.
e.g: 
Wireless -> 2001:db8:1::1
Wired -> 2001:db8:2::1